src/main/java/capstone/mfslbackend/service/PlayerService.java

```diff
@@ -77,32 +77,29 @@ public Player createPlayer(PlayerResponse playerResponse, Long teamId) {
 
             if (!player.getPosition().equals(playerResponse.getPosition())) {
                 player.setPosition(playerResponse.getPosition());
-                playerRepository.save(player);
             }
-            if (playerResponse.getNumber()!=null && !playerResponse.getNumber().equals(player.getNumber()) ) {
+            if (playerResponse.getNumber() != null && !playerResponse.getNumber().equals(player.getNumber())) {
                 player.setNumber(playerResponse.getNumber());
-                playerRepository.save(player);
             }
-            if (!player.getTeam().getTeamId().equals(teamId)){
+            if (!player.getTeam().equals(team)) {
                 player.setTeam(team);
-                playerRepository.save(player);
             }
-            return player;
+
+            return playerRepository.save(player);
```

Comment:
I was doing multiple saves, because the odds of having changes to a player are small, so it would save many db calls if we only call when there is a change. It can already take a long time for this code to run

Reply:
to confirm, would you like me to remove the saves?

Reply:
I think it was better the way it was, but if you have a good argument to keep it this way, keep it like this